securedrop/dockerfiles/xenial/python2/Dockerfile

```diff
@@ -53,8 +53,8 @@ COPY requirements requirements
 RUN pip install pip-tools && \
     pip install --require-hashes -r requirements/python2/securedrop-app-code-requirements.txt && \
     pip install -r requirements/python2/test-requirements.txt && \
-    pip install --upgrade setuptools  # Fixes #4036 pybabel requires latest version of setuptools && \
-    pip install supervisor
+    pip install supervisor && \
+    pip install --upgrade setuptools  # Fixes #4036 pybabel requires latest version of setuptools
```

Comment:
can we add a comment explaining why the order matters here?

Reply:
Did not find anything related to this in supervisor's github. I am also wondering if any dependency has issues with newer version of `setuptools`.

Reply:
Um, it was just that I'd placed the supervisor installation, which was previously done in a separate RUN step, in the command line after the pybabel/setuptools comment, so it never ran. :disappointed:

Reply:
ahhh right